backend/controllers/profile_controller.go

```
@@ -53,31 +54,34 @@ func GetProfile(ctx *gin.Context) {
+	}
+	if displayName == "" {
+		displayName = "Steve"
 	}
 
-	// Fetch leaderboard
+	// Fetch top leaderboard
 	leaderboardCursor, err := db.MongoDatabase.Collection("users").Find(
 		dbCtx,
 		bson.M{},
-		options.Find().SetSort(bson.M{"eloRating": -1}).SetLimit(10),
+		options.Find().SetSort(bson.D{{"eloRating", -1}}).SetLimit(5),
```

golangci-lint (1.64.8)

composites: go.mongodb.org/mongo-driver/bson/primitive.E struct literal uses unkeyed fields

(govet)